File: UVA/10783_Odd_Sum/solution.py
cases = int(input())

for i in range(cases):
    start = int(input())
    end = int(input())
    total = 0

    for j in range(start, end+1):
        if j % 2 == 0:
            continue
        else:
            total += j

    print(f"Case {i+1}: {total}")

File: UVA/10783_Odd_Sum/solution.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int cases;
    cin >> cases;

    for (int i = 0; i < cases; i++) {
        int start, end;
        long long total = 0;
        cin >> start;
        cin >> end;

        for (int j = start; j <= end; j++) {
            if (j % 2 == 0) { // even
                continue;
            } else { // odd
                total += j;
            }
        }

        cout << "Case " << i+1 << ": " << total << endl;
    }

    return 0;
}